quitpage.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<head>
+    <!--<link href="stylesheet.css", rel="stylesheet", type="text/css"/>-->
```

Comment:
Glad you included this! Just be sure to remove the commas after each attribute and include a space between the last attribute and closing '/'.